### PR TITLE
feat: min investment

### PR DESCRIPTION
**Pull request by tianxiuali**
# Description

min investment

# Ticket

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings

### CODE REVIEW

File: src/pages/Ethr/Withdraw/index.jsx [hunk 1]
```diff
@@ -74,7 +74,7 @@ export default function Withdraw({
       const vaultContractWithSigner = vaultContract.connect(signer)
       setCurrentStep(2)
       let tx
-      const withdrawAmount = BigNumber.from(toValue).mul(BigNumber.from(10).pow(wantTokenDecimals))
+      const withdrawAmount = BigNumber.from(BN(toValue).times(BN(10).pow(wantTokenDecimals)).toString())
```
Comment (jerrylib) [inline src/pages/Ethr/Withdraw/index.jsx:77]: times方法不对把，bignumber.js的乘法叫mutilpleBy()

File: src/pages/Usdr/Withdraw/index.jsx [hunk 2]
```diff
@@ -74,7 +74,7 @@ export default function Withdraw({
       const vaultContractWithSigner = vaultContract.connect(signer)
       setCurrentStep(2)
       let tx
-      const withdrawAmount = BigNumber.from(toValue).mul(BigNumber.from(10).pow(wantTokenDecimals))
+      const withdrawAmount = BigNumber.from(BN(toValue).times(BN(10).pow(wantTokenDecimals)).toString())
```
Comment (jerrylib) [inline src/pages/Usdr/Withdraw/index.jsx:77]: times方法不对吧，同上

**Review by jerrylib — CHANGES_REQUESTED** [review]

File: src/pages/Ethr/Withdraw/index.jsx (hunk 1 above)
Comment (tianxiuali) [reply src/pages/Ethr/Withdraw/index.jsx:77]: multipliedBy和times等价

File: src/pages/Usdr/Withdraw/index.jsx (hunk 2 above)
Comment (tianxiuali) [reply src/pages/Usdr/Withdraw/index.jsx:77]: multipliedBy和times等价

**Pull request merged**